### PR TITLE
Fix entities not being able to target island members

### PR DESCRIPTION
**Pull request by TheKiller65YT**
Fix issue described in #479 by replacing the owner check with a member check.

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/listeners/EntityTargetLivingEntityListener.java [hunk 1]
```diff
@@ -57,7 +58,7 @@ public void onEntityTargetEntity(EntityTargetLivingEntityEvent event) {
 
         // Check if the player is a guest
         User user = User.getUser(targetedPlayer);
-        if (user.islandID != island.id) {
+        if (user.getRole() == Role.Visitor) {
```
[the remark this answers is not in the document]
Comment (TheKiller65YT) [reply src/main/java/com/iridium/iridiumskyblock/listeners/EntityTargetLivingEntityListener.java:61]: Okay, then I am confused on how you get the role of a player on a certain island. The bug the user reported stems from the fact that the current version only checks if the player is the owner of the island, not just a member.

[the remark this answers is not in the document]
Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/listeners/EntityTargetLivingEntityListener.java]: it shouldnt do....
user.islandID != island.id
Checks if they are  in their island

[the remark this answers is not in the document]
Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/listeners/EntityTargetLivingEntityListener.java]: Thats not what the user reported,  they said you cant  move cows with wheat and stuff

[the remark this answers is not in the document]
Comment (TheKiller65YT) [reply src/main/java/com/iridium/iridiumskyblock/listeners/EntityTargetLivingEntityListener.java]: This event is called when the cow starts to target the player with wheat. Either way, it's still a bug fix. The initial version meant mobs only targeted the island owner and not the island members, so it's an improvement no matter what.

[the remark this answers is not in the document]
Comment (TheKiller65YT) [reply src/main/java/com/iridium/iridiumskyblock/listeners/EntityTargetLivingEntityListener.java]: @PeachesMLG

[the remark this answers is not in the document]
Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/listeners/EntityTargetLivingEntityListener.java]: Im genuinely confused as to what your trying to accomplish here, Ill look into this error in a bit

**Comment by PeachesMLG** [comment]
Fixed

**Pull request closed without merging**